Win11ThemeTest/ListBoxTest.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FlaUI.Core;
+using FlaUI.Core.AutomationElements;
+using FlaUI.Core.Definitions;
+using FlaUI.Core.Input;
+using FlaUI.UIA3;
+using FlaUI.UIA3.Identifiers;
+
+
+namespace Win11ThemeTest
+{
+    public class ListBoxTest
+    {
+        private Application app;
+        private Window window;
+        public Window listBoxWindow;
+        Button testButton;
+        ListBox listBox;
+        ListBox listBoxlength;
+        public ListBoxTest()
+        {
+            app = Application.Launch(@"..\\..\\..\\..\\TestingApplication\\bin\\Debug\\net9.0-windows\\win-x64\\TestingApplication.exe");
+
+            using (var automation = new UIA3Automation())
+            {
+                window = app.GetMainWindow(automation);
+                testButton = window.FindFirstDescendant(cf => cf.ByAutomationId("testbtn")).AsButton();
+                Mouse.Click(testButton.GetClickablePoint());
+                Wait.UntilInputIsProcessed();
+                listBoxWindow = window.FindFirstDescendant(cf => cf.ByName("ListboxWindow")).AsWindow();
+                listBox = listBoxWindow.FindFirstDescendant(cf => cf.ByAutomationId("tstLstbox")).AsListBox();
+                listBoxlength = listBoxWindow.FindFirstDescendant(cf => cf.ByAutomationId("tstlengthLstbox")).AsListBox();
+            }
+        }
+
+        //test if listbox is available in window
+        [Test]
+        public void lst1_isListboxAvailable()
```

Comment:
check if all these tests are working correctly for you - all in one go, also one by one, for me it is not running correctly.
also add window close